api/WorkerGlobalScope.json

```diff
@@ -404,7 +404,7 @@
               "version_added": "40"
             },
             "edge": {
-              "version_added": "17"
+              "version_added": "12"
```

Comment:
This and all that follows is about something other than the PR description. Should be reverted?